vagrant/provisioning/roles/upgrade-solr/tasks/main.yml

```diff
@@ -0,0 +1,65 @@
+- name: stop arkcase
+  become: yes
+  systemd:
+    daemon_reload: true
+    name: arkcase
+    state: stopped
+
+- name: copy existing solr data to backup folder
```

Comment:
Want to check existing disk space here?  If the free space is less than the Solr data folder size, this part would fail.

Reply:
You are right. Will modify accordingly.